tests/02-docker.test.js

```diff
@@ -0,0 +1,78 @@
+const { resolve } = require('path');
+
+const { exec, delay } = require('./util');
+const { evalId, containerWorkDir, defaultDelay } = require('./constants');
+
+const fs = require('fs');
+const { it } = require('@jest/globals');
+
+jest.setTimeout(30000);
+
+describe("2 - Verifica o arquivo Dockerfile", () => {
+
+  beforeAll(async () => {
+    const workDir = `${containerWorkDir}${evalId}`;
+    const volume = `-v ${resolve(evalId)}:${workDir}`;
+
+    await exec(`mkdir ${resolve(evalId)}`);
```

Comment:
```suggestion
    await exec(`rm -rf ${resolve('trybe-stranger-things-backend-*')}`)
      .catch(() => true);
    await exec('docker rm -f $(docker ps -a -f name=trybe-stranger-things-backend- -q)')
      .catch(() => true);

    await exec(`mkdir ${resolve(evalId)}`);
```
Esse recurso é interessante pra evitar que os testes vão acumulando lixo caso falhem (no meu caso aqui, quando o teste falha, o afterAll não é executado, então vale investigar se é o caso aqui)